WordPress/Classes/Services/CommentService+Likes.swift

```diff
@@ -94,22 +94,17 @@ private extension CommentService {
             return
```

Comment:
GitHub won't let me comment above this line, but line 93 is potentially an error – there's no requirement that `createNewUsers` is called from the main thread, thus `onComplete` can _sometimes_ callback on the main thread and sometimes callback on the calling thread. 

I'm not sure what to suggest, if anything – I agree with you persisting the original behaviour further down (manually specifying a queue), but I'm torn on whether we should fix this inconsistency. One approach might be `     precondition (NSThread.isMainThread, "This method must be called from the main queue")`? 

Curious about your thoughts (and whether you feel we should even be bothering to deal with it right now)

Reply:
I intentionally left this part untouched. Having looked at the code again, I feel like it should be okay to fix this inconsistency. I don't see this change would have any effect on the caller. But if it does, then it's likely something else requires a fix.